manifests/3.0.0/opensearch-3.0.0.yml

```diff
@@ -23,6 +23,15 @@ components:
     checks:
       - gradle:publish
       - gradle:properties:version
+  - name: ml-commons
+    repository: https://github.com/opensearch-project/ml-commons.git
+    ref: main
+    platforms:
+      - linux
+      - windows
+    checks:
+      - gradle:properties:version
+      - gradle:dependencies:opensearch.version: opensearch-ml-plugin
```

Comment:
Not sure if JS is one of the ml-commons's dependency? Care to make it below job-scheduler? Just to be safe?
Thanks!

Reply:
order ml-commons - js was in the manifest before it got removed last time, basically I restored manifest to the form it was before that change 5 days ago - https://github.com/opensearch-project/opensearch-build/commit/7cc32169662445fbd1d177153c2b459573cd7cb7